bundles/org.openhab.core/src/test/java/org/openhab/core/internal/scheduler/CronAdjusterTest.java

```diff
@@ -197,4 +199,13 @@ public void testCronExpression(String in, String cron, String[] outs) {
                     equalTo(out));
         }
     }
+
+    @ParameterizedTest
+    @ValueSource(strings = { "0 0 0 31 2 *", "* * *", "80 * * * * *" })
+    public void testInvalidCronExpression(String cron) {
+        assertThrows(IllegalArgumentException.class, () -> {
+            final CronAdjuster cronAdjuster = new CronAdjuster(cron);
+            cronAdjuster.adjustInto(java.time.ZonedDateTime.now());
```

Comment:
Very minor, I don't think it needs to be zoned:
```suggestion
            cronAdjuster.adjustInto(java.time.Instant.now());
```

Reply:
CronAdjuster cannot adjust an `Instant`. It will throw a `time.temporal.UnsupportedTemporalTypeException` - I tried it. So this does need to stay as `ZonedDateTime`.

Reply:
Ah, I see - thanks. I cannot resolve comment, but please consider it resolved. 🙂